Specs/DateTime/Swedish/TimeZoneExtractor.json

```diff
@@ -0,0 +1,360 @@
+[
+  {
+    "Input": "Boka ett rum åt mig Afghansk tid",
```

Comment:
Isn't this just a duplicate of the next? The only difference seems to be capitalization in the entity, which should be ignored at the model level and final result. Feel free to leave it in to show that both work.

Reply:
I temporarily added it to check that capitalization wasn't an issue. It is now removed.